application/features/user.feature

```diff
@@ -29,7 +29,7 @@ Feature: User
         | locked       | no                    |
       And the following data is not returned:
         | property                |
-        | password_hash           |
+        | password_id             |
         | password_expires_at_utc |
```

Comment:
How are we expecting the collection of password dates to come out of the API? Would it be in a `password` property?